### PR TITLE
refactor: remove password manager mention

### PR DESCRIPTION
**Pull request by AlexJupiter**
## **Description**
In the onboarding for the Secret Recovery Phrase we suggest backing up to a password manager, which we now want to change. 

This PR should now remove this bullet point mention. I've also re-ordered the previous 2nd and 3rd bullet points for this quick fix as I thought it odd to start with the advice to store the Secret Recovery Phrase in a safety deposit box. 

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/25985?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/accounts-planning/issues/529

## **Manual testing steps**

1. Onboard to MetaMask
2. Setup a new Secret Recovery Phrase

### **Before**

image

### **After**

image

## **Pre-merge author checklist**

- [x] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by coreyjanssen — DISMISSED** [review]
ok for me!

File: ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js [hunk 1]
```diff
@@ -182,9 +182,8 @@ export default function SecureYourWallet() {
           {t('seedPhraseIntroSidebarTitleTwo')}
         </Text>
         <Box as="ul" className="secure-your-wallet__list" marginBottom={4}>
+          <Text as="li">{t('seedPhraseIntroSidebarBulletTwo')}</Text>
```
Comment (danroc) [inline ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js:185]: `BulletTwo` should come after `BulletOne`, you can reverse the order or rename the variables.

**Comment by danroc** [comment]
Tested:

> image

But I left a minor comment.

File: ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js (hunk 1 above)
Comment (AlexJupiter) [reply ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js:185]: Oh I'm confused. It seems the order has been changed back from what I originally proposed. To confirm the bullets should read as follows in the following order:
- Write down and store in multiple secret places
- Store in a deposit box

Reversing the order in the HTML seemed to be the quickest way, which is what I originally did, than changing the variable names in all of the different language files.

Comment (gantunesr) [reply ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js]: Resolved in [8e27091](https://github.com/MetaMask/metamask-extension/pull/25985/commits/8e2709110332609d8bc6bb2dd2d89f1f658b57b5)

**Review by owencraston — APPROVED** [review]

**Comment by danroc** [comment]
Tested again:

image

LGTM

**Review by danroc — APPROVED** [review]

**Comment by k-g-j** [comment]
Tested in Chrome version 128.0.6613.138
Copy matches expected ✅ 
Screenshot 2024-09-12 at 4 48 54 PM

Tested in Firefox version 130.0
Copy matches expected ✅ 
Screenshot 2024-09-12 at 4 52 33 PM

**Pull request merged**